build/Common.prod.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <Import Project=".\Common.props" />
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(ForRelease)' == 'true'">
```

Comment:
where is this defined?

Reply:
It will be referenced in the Prometheus shared project.
See https://github.com/open-telemetry/opentelemetry-dotnet/pull/3430#discussion_r931522363

Reply:
Maybe we can shove the shared files under the Prometheus HttpListener project, and have the ASP.NET Core binding stealing from it?

It looks like we have a problem that is not generic, and here we try to solve it by abstraction and generalization (which doesn't seem to pay off IMHO) 😸

Reply:
would `Condition="$(MSBuildProjectName.EndsWith('.Shared'))"` work?

Reply:
@TimothyMothra, it works!